exercises/trajectories/rrt_planning.ipynb

```diff
@@ -642,11 +608,11 @@
     "\n",
     "For submission of this assignment, you must do two things. \n",
     "- Download and submit the notebook `rrt_planning.ipynb` to Gradescope's notebook submission section, along with your notebook for the other problems.\n",
-    "- Write down your answers to 7.2.a, 7.2.b, and 7.2.c to a separately pdf file and submit it to Gradescope's written submission section. \n",
+    "- Write down your answers to 7.2.a in your PDF submission to Gradescope. \n",
```

Comment:
Should be 8.2?  (in general, isn't it too fragile to list the chapter / exercise number inside these notebooks?)

Reply:
It should be, I'll correct it. (yes, its sadly true in each notebook and each time its fragile. Im normally just better at catching it)

Reply:
Will merge.  But why not just write "part (a)" here?  Does the fragile absolute reference actually add value?